src/commonMethods/webKitPerformanceCommonFunctions.js

```diff
@@ -0,0 +1,61 @@
+let samples = []
+
+/**
+ * This function resets the Samples array
+ */
+export const initializeSamplesArray = function() {
+  samples = []
+}
+
+/**
+ * This function fetches the FPS value
+ * @returns {Samples Array}
+ */
+export const fetchFPS = function() {
```

Comment:
fetchWebKitFPS, as you're only getting the FPS of the WebKit Browser instance. Not for UX/Apps/YouTube/etc.

Reply:
Updated as per the comment